src/style.css

```diff
@@ -0,0 +1,113 @@
+html,
+body,
+div {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
a lot of those properties is redundant for body and div tags, for example, fonts are inherited, if you write them once for html tag it would be enough